.travis.yml

```diff
@@ -19,6 +21,13 @@ env:
   - GCLOUD=$HOME/gcloud/google-cloud-sdk/bin 
   - CLOUDSDK_PYTHON_SITEPACKAGES=1 
   - GOOGLE_APPLICATION_CREDENTIALS=$TRAVIS_BUILD_DIR/servicekey.json
+  #GenomicsDB Stuff
+  - GATK_GENOMICSDB_BIN=src/test/resources/large/tiledb_bin
```

Comment:
So all the travis stuff is in order to run tiledb as an executable from Runtime.exec.  It's necessary to do so for the workspace / database creation.  Intel has said they're working on a write api, so hopefully it will not be necessary to have a separate binary for anything when running gatk.